src/source.ts

```diff
@@ -12,13 +12,16 @@ import { KIND_OPTIONS, CONTENT_TYPE_OPTIONS } from './constants'
 import { Cache } from './structures'
 import { Script } from './script'
 import { I18n } from './i18n'
+import { OutgoingHttpHeaders } from 'http'
```

Comment:
You need to add `body` and `headers` to the update function, otherwise it will not update.